Jenkinsfile

```diff
@@ -66,31 +100,16 @@ pipeline {
         }
       }
     }
-    stage('Code coverage') {
-      when { changeRequest() }
-      steps {
-        dir('code_coverage') {
-          sh script: "rm -r $COVERAGE_DIR || :", label: "Setup"
-          sh script: "env CC=gcc CXX=g++ cmake -DCOVERAGE=ON -DCODECOV_HTMLOUTPUTDIR=$COVERAGE_DIR $SRC/test", label: "Cmake"
-          sh script: "make -j $CPUS", label: "Make"
-          sh script: "make coverage", label: "Make coverage"
-        }
-      }
-      post {
-        success {
-           echo "Code coverage: ${env.COVERAGE_ADDRESS}/${env.JOB_NAME}"
-        }
-      }
-    }
+
     stage('Upload to bintray') {
-      when {
```

Comment:
Don't change this behaviour. We want it like it is on dev. 
when should come right after "upload to bintray" stage, no when in the latest release stage, and a "buildingTag" in the stable release.